data-explorer/kusto/query/tutorials/learn-common-operators.md

```diff
@@ -282,7 +282,7 @@ StormEvents
 
 **Output**
 
-|EpisodeId|Source|FriendlyName|
+|EventId|Source|FriendlyName|
 |---|---|
```

Comment:
```suggestion
|---|---|---|
```

The table is broken in markdown. This edit should fix that. Is it supposed to be broken ?